ui/page/livestreamSetup/view.jsx

```diff
@@ -25,11 +27,53 @@ export default function LivestreamSetupPage(props: Props) {
   const { channels, fetchingChannels, activeChannelClaim, pendingClaims } = props;
 
   const [sigData, setSigData] = React.useState({ signature: undefined, signing_ts: undefined });
+  const [showHelpTest, setShowHelpTest] = usePersistedState('livestream-help-seen', true);
+  const [spin, setSpin] = React.useState(true);
 
   const hasChannels = channels && channels.length > 0;
   const activeChannelClaimStr = JSON.stringify(activeChannelClaim);
   const streamKey = createStreamKey();
 
+  const helpText = (
+    <div className="section__subtitle">
+      <p>
+        {__(`Create a Livestream by first submitting your Livestream details and waiting for approval confirmation.`)}{' '}
+        {__(
+          `The livestream will not be visible on your channel until you are live, but you can share the URL in advance.`
+        )}{' '}
+        {__(
+          `Once the your livestream is confirmed, configure your streaming software (OBS, Restream, etc) and input the server URL along with the stream key in it.`
+        )}
+      </p>
+      <p>{__(`To ensure the best streaming experience with OBS, open settings -> output`)}</p>
+      <p>{__(`Select advanced mode from the dropdown at the top.`)}</p>
+      <p>{__(`Ensure the following settings are selected under the streaming tab:`)}</p>
+      <ul>
+        <li>{__(`Bitrate: 1000 to 2500 kbps`)}</li>
+        <li>{__(`Keyframes: 1`)}</li>
+        <li>{__(`Profile: High`)}</li>
+        <li>{__(`Tune: Zerolatency`)}</li>
+      </ul>
+      <p>
+        {__(`If using other livestreaming software, make sure the bitrate is below 5Mbps or the stream will not work.`)}
+      </p>
+      <p>
+        {__(
+          `Please note: You'll need to record your own stream through your software if you plan to share it afterward. You can also delete it if you prefer not to upload the copy. In the near future, we'll support a more streamlined process that will let you upload the replay.`
```

Comment:
remove this:
`In the near future, we'll support a more streamlined process that will let you upload the replay.`

As this PR allows you to upload the replay.